state/factory/workingset.go

```diff
@@ -603,6 +605,11 @@ func (ws *workingSet) pickAndRunActions(
 				actionIterator.PopAccount()
 				continue
 			}
+			if len(nextAction.BlobHashes()) > 0 &&
+				(!fCtx.EnableBlobTransaction || blobTxCnt >= blobTxLimit) {
```

Comment:
can remove `EnableBlobTransaction` since blob tx will be blocked earlier before hard-fork